dockerfiles/Dockerfile

```diff
@@ -51,6 +53,17 @@ RUN cd chipyard && \
         export MAKEFLAGS=-"j $(nproc)" && \
         ./scripts/build-toolchains.sh esp-tools 1>/dev/null
 
+
+# Set up FireMarshal
+SHELL ["/bin/bash", "-c"] 
```

Comment:
Can we just enforce this at the beginning of the Dockerfile instead of right when using FireMarshal? I assume most people will prefer `bash` as default instead of `sh`

Reply:
sure